### PR TITLE
bpo-37798: Fix _statistics module doc

### PR DESCRIPTION
**Pull request by corona10**
https://bugs.python.org/issue37798

### CODE REVIEW

**Comment by corona10** [comment]
@rhettinger 
I found my mistake. Module doc should be updated. Please take a look.
Backport patch needed which affected by https://github.com/python/cpython/pull/15266

**Comment by corona10** [comment]
@rhettinger ping

File: Modules/_statisticsmodule.c [hunk 1]
```diff
@@ -110,10 +110,13 @@ static PyMethodDef statistics_methods[] = {
     {NULL, NULL, 0, NULL}
 };
 
+PyDoc_STRVAR(statistics_doc,
+"statistics accelerator\n");
```
Comment (taleinat) [inline Modules/_statisticsmodule.c]:
```suggestion
"Accelerators for the statistics module.\n");
```

File: Misc/NEWS.d/next/Library/2019-08-27-17-56-51.bpo-37998.Blh9So.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Fix _statistics module doc. Patch by Dong-hee Na.
```
Comment (taleinat) [inline Misc/NEWS.d/next/Library/2019-08-27-17-56-51.bpo-37998.Blh9So.rst]: I don't think a NEWS entry is needed for a small fix to an unreleased change.

**Review by taleinat — CHANGES_REQUESTED** [review]

**Comment by corona10** [comment]
@taleinat Thank you for the review. I've updated the PR :)

**Comment by taleinat** [comment]
@corona10, great!

FYI, in the future, if you include "I have made the requested changes; please review again" in your comment, our Bedevere bot will understand, change the status labels accordingly and mention the relevant core devs.

**Review by taleinat — APPROVED** [review]
LGTM

**Pull request merged**